webcompat/static/js/lib/issues.js

```diff
@@ -383,7 +383,6 @@ issues.MainView = Backbone.View.extend({
   el: $(".js-Issue"),
   events: {
     "click .js-Issue-comment-button": "addNewComment",
-    click: "closeLabelEditor",
```

Comment:
This is where the bug was. The click handler was only bound on the `.js-Issue` element, so anything outside that would fail.